src/index.js

```diff
@@ -6,94 +6,101 @@ import { dragover } from './draggability';
-    h4.innerText = task.description
+      let btn = document.createElement('button')
+      let input = document.createElement('input')
 
-    if (task.completed) {
-      h4.classList.add('strike')
-    }
+      let h4 = document.createElement('h4')
+      h4.innerText = task.description
+      h4.contentEditable = true
+
+      h4.setAttribute('data-id', x)
+      h4.className = "x width-100%"
+
+      if (task.completed) {
+        h4.classList.add('strike')
+      }
 
-    container1.className = "flex items-center width-100%"
-    container2.className = "flex items-center space-between"
-    container3.className = "flex grow justify-end"
+      container1.className = "flex items-center width-100%"
+      container2.className = "flex items-center space-between"
+      container3.className = "flex grow justify-end"
 
-    input.className = "x clearance-right"
+      input.className = "x clearance-right"
 
-    dropArea.className = "drop-area"
-    btn.className = "bg-none material-icons border-none"
-    btn.innerText = 'more_vert'
+      dropArea.className = "drop-area"
+      btn.value = x
+      btn.className = "x bg-none material-icons border-none"
+      btn.innerText = 'more_vert'
 
-    container2.append(input, h4)
-    container3.appendChild(btn)
+      h4.addEventListener('input', e => {
+        apply(e.target.textContent, e.target.getAttribute('data-id'))
+      })
 
-    container1.append(container2, container3)
-    tasktemplate.append(dropArea, container1)
+      h4.addEventListener('click', () => {
+        del(btn, true)
+      })
 
-    dropArea.setAttribute('data-id', x)
-    input.type = 'checkbox'
-    input.setAttribute('data-id', x)
-    input.checked = task.completed
-    input.addEventListener('change', onchange)
+      h4.addEventListener('focusout', () => {
+        del(btn, false)
+      })
 
-    dropArea.addEventListener('dragover', dragover)
-    dropArea.addEventListener('dragleave', dragleave)
-    dropArea.addEventListener('dragover', dragover)
+      container2.append(input, h4)
+      container3.appendChild(btn)
 
-    dropArea.addEventListener('drop', drop);
+      container1.append(container2, container3)
+      tasktemplate.append(dropArea, container1)
+
+      dropArea.setAttribute('data-id', x)
+      input.type = 'checkbox'
+      input.setAttribute('data-id', x)
+      input.checked = task.completed
+      input.addEventListener('change', onchange)
+
+      dropArea.addEventListener('dragover', dragover)
+      dropArea.addEventListener('dragleave', dragleave)
+      dropArea.addEventListener('dragover', dragover)
+      dropArea.addEventListener('drop', drop);
+
+      g('todo_list').appendChild(tasktemplate);
+    }
 
-    g('todo_list').appendChild(tasktemplate);
+    g('clear').addEventListener('click', clear)
   }
 }
 
-onload = loadTodo();
+onload = updateIndices(() => { loadTodo() });
 /* eslint-disable no-plusplus */
 /* eslint-disable no-restricted-globals */
```

Comment:
With the ESLint inline comment, you're are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

If you are using VSCode, you can delete the inline comment and hover over the error(configure Eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇.

Reply:
Hi @bellom, I have made the required changes. Please check.